apps/tup-ui/src/main.tsx

```diff
@@ -5,6 +5,7 @@ import ReactDOM from 'react-dom/client';
 import App from './App';
 
 import './main.global.css';
+import './main.global.for-core-styles.css';
```

Comment:
**What are you doing, Wes…** I took a shortcut and made a programmatic TODO. I accept complaints and will move now if requested.